src/game.cpp

```diff
@@ -3558,13 +3556,18 @@ void Game::updateSound(f32 dtime)
 			      camera->getDirection(),
 			      camera->getCameraNode()->getUpVector());
 
-	// Check if volume is in the proper range, else fix it.
-	float old_volume = g_settings->getFloat("sound_volume");
-	float new_volume = rangelim(old_volume, 0.0f, 1.0f);
-	sound->setListenerGain(new_volume);
+	bool mute_sound = g_settings->getBool("mute_sound");
+	if (mute_sound)
+		sound->setListenerGain(0.0f);
+	else {
```

Comment:
Add brackets for this upper case. Technically you're right by leaving them away but the code style says something else.

Reply:
I can do that if that's preferred, but could you show me where they say so?

Reply:
http://dev.minetest.net/Code_style_guidelines : 
> The coding style is based on the [Linux kernel code style](https://github.com/torvalds/linux/blob/master/Documentation/process/coding-style.rst).

Chapter 3.0, last code example + description. It also states that the bracket belongs into the line of the `else` keyword. i.e. `} else {`.

Reply:
Thank you, done and rebased.